courses/elixir.html

```diff
@@ -255,7 +261,7 @@ <h5 class="uppercase">Databases</h5>
                                           2. MySQL<br>
                                           3. Using ORMs like Sequelize<br>
                                           4.MongoDB<br>
-                                          
```

Comment:
these whitespace fixes are valid, but I'd recommend editing whitespace (or any other indentation related improvements) in separate commits).